client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -123,8 +115,18 @@ class StoreStats extends Component {
 					/>
 				</StatsPeriodNavigation>
 				<div className="store-stats__widgets">
-					{ widgetList1 }
-					{ widgetList2 }
+					<div className="store-stats__widgets-column spark-widgets" key="sparkwidgets">
+						<Module
+							siteId={ siteId }
+							header={ null }
+							emptyMessage={ translate( 'No data found.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'No data found' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wcss-refactor-statsorders)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).